core/src/main/java/com/exadel/etoolbox/backpack/core/dto/response/ResourceRelationships.java

```diff
@@ -0,0 +1,60 @@
+package com.exadel.etoolbox.backpack.core.dto.response;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Represents the wrapper for a collection of active paths and broken paths obtained as a result of the search.
+ */
+public class ResourceRelationships {
+
+    private List<String> actualPaths;
+    private List<String> brokenPaths;
+
+    public ResourceRelationships(List<String> actualPaths, List<String> brokenPaths) {
+        this.actualPaths = actualPaths != null ? actualPaths : new ArrayList<>();
+        this.brokenPaths = brokenPaths != null ? brokenPaths : new ArrayList<>();
+    }
+
+    public List<String> getActualPaths() {
+        return actualPaths;
+    }
+
+    public void setActualPaths(List<String> actualPaths) {
+        this.actualPaths = actualPaths != null ? actualPaths : new ArrayList<>();;
+    }
+
+    public List<String> getBrokenPaths() {
+        return brokenPaths;
+    }
+
+    /**
+     * Concatenation two ResourceRelationships objects into one
```

Comment:
(cosmetic) "Concatenation OF two... "
Or rather, "ConcatenateS two...". Usually it starts with a verb.
Also, why "of two objects" if there is only one argument?) I'd say "Appends the provided {@link ResourceRelationships} object to the current one"